skipruntime-ts/server/src/rest.ts

```diff
@@ -13,9 +13,15 @@ export function controlService(service: ServiceInstance): express.Express {
 
   // Streaming control API.
   app.post("/v1/streams/:resource", (req, res) => {
+    const session_id = req.header("Session-Id") ?? null;
```

Comment:
Could we document this (and other endpoints which read the `Session-Id` header) in `www/docs/resources.md`?